exchange/bidder_validate_bids.go

```diff
@@ -116,7 +116,10 @@ func validateBid(bid *entities.PbsOrtbBid) (bool, error) {
 		return false, fmt.Errorf("Bid \"%s\" missing required field 'impid'", bid.Bid.ID)
 	}
 	if bid.Bid.Price < 0.0 {
-		return false, fmt.Errorf("Bid \"%s\" does not contain a positive (or zero if there is a deal) 'price'", bid.Bid.ID)
+		if debug {
+			return false, fmt.Errorf("Bid \"%s\" does not contain a positive (or zero if there is a deal) 'price'", bid.Bid.ID)
+		}
+		return false, nil
 	}
 	if bid.Bid.Price == 0.0 && bid.Bid.DealID == "" {
 		return false, fmt.Errorf("Bid \"%s\" does not contain positive 'price' which is required since there is no deal set for this bid", bid.Bid.ID)
```

Comment:
I think the `debug` flag also applies to this block. The Java version has the same error message for price < 0 and when price is 0 with no deal ID. We can still have different error messages, as I think ours are more informative, but we should still only show them when `debug` is enabled.